eng/build.ps1

```diff
@@ -258,7 +258,7 @@ if ($os -eq "Browser") {
   # override default arch for Browser, we only support wasm
   $arch = "wasm"
 
-  if ($msbuild -eq $True) {
+  if ($msbuild -eq $True -and $runtimeFlavor -ne "CoreCLR") {
     Write-Error "Using the -msbuild option isn't supported when building for Browser on Windows, we need need ninja for Emscripten."
```

Comment:
nit: double need

Reply:
i mean to say that in the statement

>  we need need ninja for Emscripten

need should be once.  A minor typo. I found while reading the code.

Reply:
Thanks, its not part of this change, and its upstream in the runtime repo (https://github.com/dotnet/runtime/blob/72d643d05ab23888f30a57d447154e36f979f3d1/eng/build.ps1#L276).  Should be fixed there really.